src/csharp/Grpc.Core/CallOptions.cs

```diff
@@ -203,6 +216,17 @@ public CallOptions WithCredentials(CallCredentials credentials)
         }
 
         /// <summary>
+        /// Returns a new instance of <see cref="CallOptions"/> with <c>Items</c> set to the value provided.
+        /// Values of all other fields are preserved.
```

Comment:
Possibly clarify that any *previous* items are effectively lost rather than merged?